oda_api/api.py

```diff
@@ -838,6 +838,11 @@ def get_product(self,
         submit query, wait (if allowed by self.wait), decode output when found
         """
 
+        self.logger.warning('please beware that by default, in a typical setup, oda_api will not output much. '
+                            'To learn how to increase the verbosity, please refer to the documentation: '
+                            'https://oda-api.readthedocs.io/en/latest/user_guide/ScienceWindowList.html?highlight=logging#Let\'s-get-some-logging'
+                            )
+
```

Comment:
Because constructor never produces a message user really expects.

Instead, it is during `get_product` that user wants to see the feedback. It's a long call, and it is when the messages may be, in some cases, useful.

Constructor may be called once way above. Some other log may happen in after.
And then get_product will be still empty.

The communication is expected at `get_product` and that's where the warning should happen.